samples/quarkus/pet-store/sam.native.yaml

```diff
@@ -0,0 +1,36 @@
+  AWSTemplateFormatVersion: '2010-09-09'
+  Transform: AWS::Serverless-2016-10-31
+  Description: AWS Serverless Quarkus HTTP - com.amazon.quarkus.demo::pet-store
+  Globals:
+    Api:
+      EndpointConfiguration: REGIONAL
+      BinaryMediaTypes:
+        - "*/*"
+
+  Resources:
+    PetStoreNativeFunction:
+      Type: AWS::Serverless::Function
+      Properties:
+        Handler: not.used.in.provided.runtimei
+        Runtime: provided
+        CodeUri: target/function.zip
+        MemorySize: 128
+        Policies: AWSLambdaBasicExecutionRole
+        Tracing: Active
+        Timeout: 15
+        Environment:
+          Variables:
+            DISABLE_SIGNAL_HANDLERS: true
```

Comment:
What does this do? Is it worth mentioning it in the readme?

Reply:
This is mentioned in Quarkus doc and a github issue. 
https://github.com/quarkusio/quarkus/issues/4262#issuecomment-537599842

It seems custom runtime does not allow the syscalls required by the GraalVM signals implementation. 

I added this in the readme.